docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -133,15 +125,28 @@ Alternatively, you can pass in `--starter` to `odo create` to have odo download
 
 . Create a URL in order to access the deployed component:
 +
+* Run below for Kubernetes cluster
```

Comment:
Maybe simply `For Kubernetes cluster`?

Reply:
Honestly, I don't think this matters much... We could actually keep this the same and just remove `--host` as by default it will use the default hostname? Or add a small note saying: `If using Kubernetes, you have to pass in --host`